ncclient/devices/ciena.py

```diff
@@ -0,0 +1,24 @@
+from .default import DefaultDeviceHandler
+from ncclient.xml_ import BASE_NS_1_0
+
+
+class CienaDeviceHandler(DefaultDeviceHandler):
+    """
+    Ciena handler for device specific information.
+    """
+
+    def __init__(self, device_params):
+        super(CienaDeviceHandler, self).__init__(device_params)
+
+    def get_capabilities(self):
+        return [
+            "urn:ietf:params:netconf:base:1.0",
+        ]
```

Comment:
This has the effect of restricting what capabilities are offered by `ncclient` to a Ciena device. Is that your intention? If so, it is not actually necessary.

Reply:
Thanks. I will remove.